generators/summarization/sumy_website_summarizer/README.md

```diff
@@ -0,0 +1 @@
+This brick module is using the sumy library from GitHub to summarize texts that were scraped from websites like wikipedia or blog articles. The function excepts an url to a website and not a text itself. It uses Latent Semantic Analysis to extract structures of the words and texts used to find relevant sentences. You can read more about the sumy libraray here: https://github.com/miso-belica/sumy
```

Comment:
I think "that were scraped from websites" implies that this process has already been done by sumy, but you query the website first, right? sumy does not work offline, correct?

Reply:
Correct!

Reply:
I now added a sentence that the articles itself are scraped by sumy and that an internet connection is needed.